ext/dav1d.cmd

```diff
@@ -3,22 +3,17 @@
 
 : # The odd choice of comment style in this file is to try to share this script between *nix and win32.
 
-: # meson and ninja must be in your PATH.
+: # meson must be in your PATH.
 
 : # If you're running this on Windows, be sure you've already run this (from your VC2019 install dir):
 : #     "C:\Program Files (x86)\Microsoft Visual Studio\2019\Professional\VC\Auxiliary\Build\vcvars64.bat"
 
 : # When updating the dav1d version, make the same change to dav1d_android.sh.
 git clone -b 1.5.1 --depth 1 https://code.videolan.org/videolan/dav1d.git
 
-cd dav1d
-mkdir build
-cd build
-
 : # macOS might require: -Dc_args=-fno-stack-check
 : # Build with asan: -Db_sanitize=address -Db_lundef=false
 : # Build with msan: -Db_sanitize=memory -Db_lundef=false -Denable_asm=false
 : # Build with ubsan: -Db_sanitize=undefined -Db_lundef=false
-meson setup --default-library=static --buildtype release -Denable_tools=false -Denable_tests=false ..
-cd ../..
-ninja -C dav1d/build
+meson setup --default-library=static --buildtype release -Denable_tools=false -Denable_tests=false dav1d/build dav1d
+meson compile -C dav1d/build
```

Comment:
Optional: I think it is fine to use the `ninja -C dav1d/build` command as before. The instructions in the dav1d website also use the ninja command:
https://code.videolan.org/videolan/dav1d#compile
https://code.videolan.org/videolan/dav1d#build-documentation

Reply:
I guess if we use `cmake` instead of `ninja`, we may as well use `meson` instead of `ninja`? Not sure I understand the nuance.

Reply:
It is fine to use `meson compile`. I am not that familiar with meson. I was just pointing out that the official dav1d build instructions still use `ninja`.